src/main/java/ToDoTask.java

```diff
@@ -0,0 +1,15 @@
+public class ToDoTask extends Task {
```

Comment:
I had that initially, but it would trigger the "todo" feature (the one that makes text turn light blue) whenever I mention it in a comment, so I decided to add the word task behind.